src/agent/src/Bridge/Brave/Brave.php

```diff
@@ -75,7 +75,11 @@ public function __invoke(
         }
 
         return array_map(static function (array $result) {
-            return ['title' => $result['title'], 'description' => $result['description'], 'url' => $result['url']];
+            return [
+                'title' => $result['title'] ?? '',
+                'description' => $result['description'] ?? '',
+                'url' => $result['url'] ?? '',
```

Comment:
All of them are optional? If yes I think it does not make sense to return an array where all values can be an empty string.